Header header
float32 drone_timeuint16 tag
uint16 size
uint32 nb_detected
uint32[] type
uint32[] xc
uint32[] yc
uint32[] width
uint32[] height
uint32[] dist
float32[] orientation_angle
ardrone_autonomy::matrix33[] rotation
ardrone_autonomy::vector31[] translation
uint32[] camera_source
